data/pirate jobs.txt

```diff
@@ -0,0 +1,2766 @@
+	repeat
+	job
+	description "A Syndicate fleet near the <system> system has halted our raids. Destroy them and return to <planet> for a payment of <payment>."
+	to offer
+		random < 15
+		"combat rating" > 750
+	source
+		government "Pirate"
+		attributes "core pirate"
+	npc kill
+		personality heroic staying
+		government "Syndicate"
+		system
+			distance 1 3
+		fleet "Large Syndicate"
+		fleet "Small Syndicate"
+		fleet "Small Syndicate"
+		dialog "The fleet has been eliminated. You can claim the bounty payment by returning to <destination>."
+	on complete
+		payment 400000
+		dialog "The pirates of <planet> gratefully pay you <payment> for eliminating the Syndicate fleet."
+
+mission "Elminating Law Enforcement (South, Small) [0]"
+	name "Militia near <system>"
+	repeat
+	job
+	description "A Militia fleet near the <system> system has halted our raids. Destroy them and return to <planet> for a payment of <payment>."
+	to offer
+		random < 25
+		"combat rating" > 500
+	source
+		government "Pirate"
+		attributes "south pirate"
+	npc kill
+		personality heroic staying
+		government "Militia"
+		system
+			distance 1 3
+		fleet "Small Militia"
+		fleet "Small Militia"
+		dialog "The fleet has been eliminated. You can claim the bounty payment by returning to <destination>."
+	on complete
+		payment 250000
+		dialog "The pirates of <planet> gratefully pay you <payment> for eliminating the Militia fleet."
+
+mission "Elminating Law Enforcement (South, Large) [1]"
+	name "Militia near <system>"
+	repeat
+	job
+	description "A Militia fleet near the <system> system has halted our raids. Destroy them and return to <planet> for a payment of <payment>."
+	to offer
+		random < 15
+		"combat rating" > 750
+	source
+		government "Pirate"
+		attributes "south pirate"
+	npc kill
+		personality heroic staying
+		government "Militia"
+		system
+			distance 1 3
+		fleet "Large Militia"
+		fleet "Small Militia"
+		fleet "Small Militia"
+		dialog "The fleet has been eliminated. You can claim the bounty payment by returning to <destination>."
+	on complete
+		payment 400000
+		dialog "The pirates of <planet> gratefully pay you <payment> for eliminating the Militia fleet."
+
+mission "Eliminating Competition (North, Small) [0]"
+	name "Competition near <system>"
+	description "A small rival gang is causing trouble near the <system> system. Take them out and return to <planet> for a payment of <payment>."
+	repeat
+	job
+	to offer
+		random < 20
+		"combat rating" > 350
+	source
+		government "Pirate"
+		attributes "north pirate"
+	npc kill
+		personality heroic staying
+		government Bounty
+		system
+			distance 1 2
+		fleet "Small Northern Pirate"
+		fleet "Small Northern Pirate"
+		fleet "Small Northern Pirate"
+		fleet "Small Northern Pirate"
+		dialog "The fleet has been eliminated. You can claim the bounty payment by returning to <destination>."
+	on complete
+		payment 500000
+		dialog "The pirates of <planet> gratefully pay you <payment> for eliminating 
```

Comment:
Readded the bounty governments so that these jobs work as well as making normal governments attack you if you attack the bounty governments as to go along with MZ's desires.